native-image-workshop/README.md

```diff
@@ -1,51 +1,59 @@
 # GraalVM Native Image Workshop
 
+## Prerequisites
+
+* OS : Linux or Mac OSX
+* Apache Maven
+
 ## Introduction
 
-This workshop aims to walk you through a number of features GraalVM's Native Image offers,
-namely:
+We are going to use a fairly trivial application to walk through using GraalVM Native Image. Along the way we will see:
 
-1. Run a basic Java app
-2. Build a native image from it and see how that compares in terms of startup time
-3. Look at how we use the tracing agent to identify any dynamic features
-4. Run some of the application at the native image build time
+1. How you can turn a Java app into a native executable
+2. How you can build a GraalVM Native IMage that works with the dynamic parts of Jva
```

Comment:
> How you can build a native image that works with the dynamic parts of Java

Reply:
Not sure how that skipped through. Updating